src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -158,9 +158,19 @@ export const WorkspaceData = (signal) => ({
     const res = await fetchWDS(root)('status', _.merge(authOpts(), { signal }));
     return res.json();
   },
-  listInstances: async (root: string): Promise<any> => {
-    const res = await fetchWDS(root)('instances/v0.2', _.merge(authOpts(), { signal }));
-    return res.json();
+  listInstances: async (root: string, workspaceId: string): Promise<any> => {
```

Comment:
is it worth renaming to `listCollections` while we're in here?

Reply:
Ah yeah I was going to do that but forgot

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/4992/commits/f74f9e9b4565717b128556d946e7aa73eabc864c